x/bep3/types/params.go

```diff
@@ -18,15 +18,15 @@ var (
 	AbsoluteMinimumBlockLock int64 = 50
 	DefaultMinBlockLock      int64 = 80
 	DefaultMaxBlockLock      int64 = 600
-	DefaultSupportedAssets         = AssetParams{AssetParam{Denom: "kava", CoinID: "459", Limit: sdk.NewInt(1), Active: false}}
+	DefaultSupportedAssets         = AssetParams{AssetParam{Denom: "kava", CoinID: 459, Limit: sdk.NewInt(1), Active: false}}
```

Comment:
do we want denom `ukava` here?

Reply:
This is just a placeholder, "kava"/"ukava" isn't used anywhere - just "bnb" is transfered back and forth. We should actually probably change this to "bnb", mark it as active, and update the limit to be the expected asset limit at launch.

Reply:
https://github.com/Kava-Labs/kava/pull/397/commits/6eea7377ae5b59fa2b7a13b6acd998cd71d16c75